src/session.test.js

```diff
@@ -211,14 +216,14 @@ describe('Session', () => {
 
     describe('#signUp()', () => {
       it('updates the visitor id in the cookie', done => {
-        $.cookie = jest.fn();
+        Cookies.set.mockClear();
```

Comment:
Isn't there a thing where we can define an after all that will reset all the mocks in jest? And we can do that across the whole suite? I find resetting mocks in individual specs to be an anti pattern.

Reply:
We have the `jest.config.js` set up to clear mocks after every test, this call here clears it after the session `initialize` method call. I've moved it into the `beforeEach` setup hook.